### PR TITLE
Stylelinter Added to Workflows.

### PR DESCRIPTION
**Pull request by andyderek**
The Stylelinter is running with default configs.

---
This change is [Reviewable](https://reviewable.io/reviews/xcov19/mycovidconnect/81)

### CODE REVIEW

File: .github/workflows/csslint.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+jobs:
+    linters:
+        name: stylelint
+        runs-on: ubuntu-latest
+      
+        steps:
+            - uses: actions/checkout@v2
+            - uses: actions-hub/stylelint@master
+        env: 
+            PATTERN: "**/*.css"
```
Comment (codecakes) [inline .github/workflows/csslint.yml:13]: add a line End of file;

File: .github/workflows/csslint.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+name: Stylesheet Lint
+on: [push, pull_request]
+
+jobs:
```
Comment (codecakes) [inline .github/workflows/csslint.yml:4]: make tab 2 space not 4

**Review by codecakes — CHANGES_REQUESTED** [review]
looks good. minor changes ive requested then you can merge

File: .github/workflows/csslint.yml (hunk 1 above)
Comment (andyderek) [reply .github/workflows/csslint.yml]: Ok I added a line at the end of the file.

File: .github/workflows/csslint.yml (hunk 2 above)
Comment (andyderek) [reply .github/workflows/csslint.yml:4]: I changed my tabs to 2 spaces instead of 4.

**Review by codecakes — APPROVED** [review]

**Pull request merged**